src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -276,7 +276,7 @@ public function iSetItsMetaDescriptionToTooLongStringIn(string $localeCode): voi
     public function iSetItsNonTranslatableAttributeTo(ProductAttributeInterface $attribute, string $value): void
     {
         $this->client->addSubResourceData(
-        'attributes',
+            'attributes',
```

Comment:
These changes should be done in a separate PR as they are not related

Reply:
Maybe let's ourselves to introduce small improvements. Creating a PR for adding an indent seems like being a little bit too orthodox for me :x. It's not a big deal to review a few more lines, especially when it's noticeable it's not a business logic.